.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,42 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+<!-- In the following, please describe your issue in detail! -->
+<!-- If some of the sections do not apply, just remove them. -->
+
+### Short description
+<!-- This should summarize the issue. -->
+
+### Code to reproduce
```

Comment:
Maybe instead of the code comment below add a template comment like

> Please provide a minimal working example that reproduces the issue in the code block below.
> Ideally, this should be a full example someone else could copy+paste and run without additional setup.